### DISCUSSION TITLE
v7 benchmarks and bundle size

### DISCUSSION THREAD

**Discussion by b0iq** [mantinedev/mantine#4797] [Q&A]
👋 Hey,
first I’d like to thank all the contributors who produce this awesome lib. 

I have question regarding v7.

I want to determine the performance improvements in numbers (benchmarks)

Also what is the bundle size difference?

I want to know if it worth the time to upgrade my projects…

**Comment by rtivital** [comment]
https://bundlephobia.com/package/@mantine/core@7.0.0
